README.md

```diff
@@ -17,73 +20,89 @@ Add `tailwind_formatter` to your list of dependencies in `mix.exs`:
 ```elixir
 def deps do
   [
-    {:tailwind_formatter, "~> 0.3.0"}
-
-    # alternatively, keep track with the latest release:
-    {:tailwind_formatter, github: "100phlecs/tailwind_formatter"}
+    {:tailwind_formatter, "~> 0.3"}
```

Comment:
Reasoning for this change:

- I don't think the hint to use the :github source is needed (if you like to be bleeding-edge, you'll already know this is possible 😄)
- I think it's okay to be a bit lax on the version specification recommendation for a formatter

Reply:
Another thought -- should it be recommended to specify the dependency like this instead?

```elixir
{:tailwind_formatter, "~> 0.3", only: :dev, runtime: false}
```

This is how I personally installed it in my project and it seems to be working great!

Reply:
I think specifying the dependency as `:dev` only makes sense if you want to make that change :)